src/fmi4c_private.h

```diff
@@ -252,6 +253,7 @@ typedef struct {
 } fmi2DataMe_t;
 
 typedef struct {
+    const char* fmiVersion_;
```

Comment:
Is there a reason for the underscore?

Reply:
@lochel i guess there is no particular reason, but i see there is also model version, so just in case i added the underscore

Reply:
Perhaps "version" should be renamed to "modelVersion" instead to make it more clear. But that's not so important.